frappe/core/doctype/sms_settings/sms_settings.py

```diff
@@ -18,6 +18,9 @@ class SMSSettings(Document):
 def validate_receiver_nos(receiver_list):
 	validated_receiver_list = []
 	for d in receiver_list:
+		if not d:
+			break
```

Comment:
continue?

Reply:
It will silently break, user wouldn't know that there was an issue.

Reply:
It does raise an exception after that line, I am using existing code for it.